posthog/api/feature_flag.py

```diff
@@ -56,6 +59,15 @@ def update(self, instance: FeatureFlag, validated_data: Dict, *args: Any, **kwar
         except IntegrityError:
             raise serializers.ValidationError("This key already exists.", code="key-exists")
 
+    def perform_destroy(self, key: str) -> bool:
```

Comment:
Oops, looks like some accidental code!

Reply:
You right, it came from other commit. I just reverted it